css-conditional-5/scroll_state_explainer.md

```diff
@@ -192,6 +231,43 @@ Query values for `stuck`:
 Query values for `snapped`:
 
 - `none`
+- `x`
+- `y`
+- `block`
+- `inline`
+- `both`
+
+Query values for `scrollable`:
+
+- `none`
+- `top`
+- `right`
+- `bottom`
+- `left`
+- `block-start`
+- `inline-start`
+- `block-end`
+- `inline-end`
+- `x`
+- `y` 
+- `block`
+- `inline`
+
+
+Query values for `direction`:
+
+- `none`
```

Comment:
Should `none` be removed? Or is it still expected that the start value is `none` even for the state that reflects the most recent scroll?